base_attachment_object_storage/models/ir_attachment.py

```diff
@@ -87,14 +86,28 @@ def _save_in_db_anyway(self):
         if self.res_model == 'ir.ui.view':
             # assets are stored in 'ir.ui.view'
             return True
-
-        # Binary fields
-        if self.res_field:
-            # Binary fields are stored with the name of the field in
-            # 'res_field'
-            # 'image' fields can be rather large and should usually
-            # not be requests in bulk in lists
-            if self.res_field and self.res_field in self._local_fields:
+        import pdb
+        pdb.set_trace()
+        # Check if model must never be stored on DB
+        excluded_model_settings = self.env['ir.config_parameter'].sudo().\
+            get_param('excluded.models.storedb', default='')
+        excluded_model_for_db_store = excluded_model_settings.split(',')
+        if any(self.res_model == val for val in
+               excluded_model_for_db_store):
```

Comment:
`if self.res_model in excluded_model_for_db_store:`?

Reply:
yep much better

Reply:
Friday copy/paste :) , I have update it